developers/blobstream.md

```diff
@@ -14,79 +16,106 @@ on Ethereum, for integration by developers into L2 contracts. This enables Ether
 the first with Data Availability Sampling (DAS).
 
-The following docs go over how developers can integrate Blobstream.
-There are also docs on how to run a [Blobstream orchestrator](../nodes/blobstream-binary.md)
-as a Celestia validator which won't be covered in the following sections
-aimed at developers.
-
-## Overview
-
-Blobstream,
-consists of two components: an [orchestrator](../nodes/blobstream-orchestrator.md)
-and a [relayer](../nodes/blobstream-relayer.md).
-
-In the following diagram, we show how a layer 2 (L2) would post data to
-Celestia and then verify that it was published in the target EVM chain.
-
-![Blobstream-Architecture](/img/blobstream/Blobstream.png)
-
-Data will first be attested to by the Celestia validator set, _i.e._
-signing commitments committing to the data. Then, these signatures will be
-relayed to the target EVM chain (in this case, Ethereum). Finally,
-the L2, or any party, will be able to verify that the data was published
-to Celestia directly on the EVM chain on the Blobstream smart contract. You can
-reference [the Blobstream smart contract](https://github.com/celestiaorg/blobstream-contracts/blob/master/src/Blobstream.sol).
-
-The **orchestrator** is part of the validator setup and works as follows:
-
-- celestia-app creates an attestation on the state machine level that needs to
-  be signed
-- The orchestrator queries the attestation, signs it, then submits the signature
-  to the Blobstream P2P network
-
-The **relayer** submits the attestations' signatures from the Blobstream
-P2P network to the target EVM chain.
+An implementation of Blobstream, by [Succinct](https://docs.succinct.xyz/), called
+[Blobstream X](https://github.com/succinctlabs/blobstreamx), is out
+and will be used in our canonical deployments. This implementation proves the
+validity of Celestia block headers on a target EVM chain using zero-knowledge
+proofs, which allow inheriting all the security
+guarantees of Celestia.
+
+## What is Blobstream X?
+
+Blobstream X is an implementation of Blobstream with a
+ZK light client that bridges Celestia’s modular DA layer to
+Ethereum to allow high-throughput rollups to use Celestia’s DA while settling
+on Ethereum.
+
+Optimistic or ZK rollups that settle on Ethereum but wish to use Celestia for
+DA require a mechanism for _bridging_ Celestia’s data root to Ethereum as part
+of the settlement process. This data root is used during inclusion proofs to
+prove that particular rollup transactions were included and made available in
+the Celestia network.
+
+Bridging Celestia’s data root to Ethereum requires running a Celestia
+_light client_ as a smart contract on Ethereum, to make the latest state
+of the Celestia chain known on Ethereum and available to rollups. Blobstream
+X utilizes the latest advances in zero-knowledge proofs to generate a
+_succinct proof_ that enough Celestia validators have come to consensus
+(according to the Tendermint consensus protocol) on a block header, and
```

Comment:
In the section further down the page "Blobstream vs. DACs" there is a reference to CometBFT. What should we use as standard in the docs @rach-id ?

Reply:
cometBFT